src/components/reservation/helper.ts

```diff
@@ -251,4 +253,56 @@ export const getInstantLoanBranchHoldingsAboveThreshold = (
     ({ materials }) => materials.length >= Number(instantLoanThresholdConfig)
   );
 
+export const removePrefixFromBranchId = (branchId: string) => {
+  const splitBranchId = branchId.split("-");
+  return splitBranchId[1];
+};
+
+export const getPatronAddress = (address: AddressV2) =>
```

Comment:
This is very global. Do we maybe have a patron helper file in core where we could place this instead?